.gitignore

```diff
@@ -0,0 +1 @@
+.project
```

Comment:
.project are Eclipse project files.
Ignored, since the IDE is a preference of the developer, not the project.